CHANGELOG.markdown

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+* Added `svgo-allow-lossy` and `svgo-precision` options to use svgo in lossy mode. This sets svgo `precision`, which can result in substantially smaller svgs. Lower values are more lossy. 3 is the default, but many SVGs will work well even with 0 or 1. [#210] [@gurgeous](https://github.com/gurgeous)
```

Comment:
I think it is better to use complete option names `--svgo-precision`, also I assume default is to use global `--allow-lossy` instead of per worker `--xxx-allow-lossy`.
And please create complete link to #210, so it works even outside of github and please add a link to this PR too.